processing/src/main/java/io/druid/query/topn/TopNParams.java

```diff
@@ -43,10 +43,6 @@ protected TopNParams(
     this.cursor = cursor;
     this.cardinality = selectorPlus.getColumnSelectorStrategy().getCardinality(selectorPlus.getSelector());
     this.numValuesPerPass = numValuesPerPass;
-
```

Comment:
This is removed since the TopNParams can now hold CARDINALITY_UNKNOWN for certain dimension selectors